src/translator_nl.h

```diff
@@ -18,7 +18,7 @@
 #ifndef TRANSLATOR_NL_H
 #define TRANSLATOR_NL_H
 
-class TranslatorDutch : public Translator
+class TranslatorDutch : public TranslatorAdapter_1_9_5
 {
   public:
```

Comment:
For the dutch version of the translation of Flowchart you can use: Stroomschema so
```
    virtual QCString trFlowchart()
    { return "Stroomschema: "; }
```

Reply:
For the dutch translation of  `virtual QCString trMemberFunctionDocumentation()`
```
    virtual QCString trMemberFunctionDocumentation()
    {
      if (Config_getBool(OPTIMIZE_OUTPUT_VHDL))
      {
        return "Documentatie van functie/procedure/process members";
      }
      else
      {
        return ""Documentatie van functie members"";
      }
    }
```

Reply:
Done